TravelPoints-UI/src/app/app.routes.ts

```diff
@@ -1,10 +1,12 @@
 import {RouterModule, Routes} from '@angular/router';
 import {NgModule} from "@angular/core";
 import {HomePageComponent} from "./components/home-page/home-page.component";
+import {TouristAttractionPageComponent} from "./components/tourist-attraction-page/tourist-attraction-page.component";
 
 export const routes: Routes = [
   {path: '', component: HomePageComponent},
```

Comment:
S: the default route MUST always be the last one on the list. If not, there is a big change for routing bugs because the default route is matched by everything